great-idea/index.html

```diff
@@ -15,9 +15,54 @@
 </head>
 
   <body>
+<div class="container">
+  <a href='#'>Services</a>
+  <a href='#'>Product</a>
+  <a href='#'> Vision</a>
+  <a href='#'> Features </a>
+  <a href='#'>About</a>
+  <a href='#'>Contact </a>
+</div>
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+ <header>
+  Innovation
+  On
+  Demand
+  </header>
+  <button> Started </button>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+  <img src="img/header-img.png" alt="Image of a code snippet.">
+<
+ <div class="para1"> 
+   <h3>Features</h3>
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</p>
+
+  <h3>About</h3>
+  <p> elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</p>
+</div>
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  <div class="para2">
+  <h3>Services</h3>
+<p>  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</p>
+
+
+  <h3>Product</h3>
+  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
```

Comment:
This is what I call floating content. This is a bad practice. The best way to think of it is that all content needs to be wrapped in an HTML tag.